janus_server/src/task.rs

```diff
@@ -323,6 +327,14 @@ impl Task {
     }
 }
 
+fn fmt_vector_of_urls(urls: &Vec<Url>, f: &mut Formatter<'_>) -> fmt::Result {
```

Comment:
I sorta don't like that this duplicates code with the same function in `janus_client`'s lib.rs, but it's simple enough that I don't think this warrants what would be necessary to provide it to both w/o also exposing it publicly from our published crates (i.e. creating a new "utility" unpublished crate that both crates could rely on).